README.md

```diff
@@ -22,6 +22,18 @@ Afterwards clone the repo and install it:
 This will install `osmtogtfs.py` executable on your OS. You can also directly run the script found
 in the source code. Make sure to run it with python 3.
 
+# Tests
+We use wonder `pytest` package for testing. Install pytest and run the tests:
```

Comment:
@mehdisadeghi you mean "wonderful"? typo?

Reply:
Haha, yeah!